adapters/src/main/kotlin/com/krzykrucz/elesson/currentlesson/attendance/AttendanceWorkflowAdapter.kt

```diff
@@ -0,0 +1,21 @@
+package com.krzykrucz.elesson.currentlesson.attendance
+
+import org.springframework.context.annotation.Bean
+import org.springframework.context.annotation.Configuration
+import org.springframework.http.MediaType
+import org.springframework.web.reactive.function.server.router
+
+@Configuration
+class AttendanceWorkflowAdapter {
+
+    @Bean
+    fun attendanceRouter() = router {
+        (path("/attendance") and accept(MediaType.APPLICATION_JSON)).nest {
+            POST("/absent", handleNoteAbsentRequest())
+            POST("/present", handleNotePresentRequest())
```

Comment:
it should work more like the following:
1. POST /present with a request body containing only the checked student and a lesson identifier
2. fetch NotCompletedAttendance from DB (if doesn't exist then a StartedLesson from the prev. workflow)
3. compute the attendance
4. persist the attendance in DB, no matter if it finished or not
5. respond with current attendance state (it's an optional) and a boolean indicating if the check is complete

There should be no endpoint for finishing the attendance

There shouldn't be more POST endpoints than blue post-its in the design-level Event Storming or more GET endpoints than green post-its 😃

Reply:
Oh ok, I get it now :D